swiss-ng-api/README.md

```diff
@@ -11,39 +11,40 @@ spezification:
+| purposeCode                             | purposeCode                       | optional                            | optional                                        | optional                           | optional                                | optional                           | optional                            |
+| remittanceInformation UnstructuredArray | Max140Text                        | n.a.                                | n.a.                                            | n.a.                               | n.a.                                    | n.a.                               | n.a.                                |
+| serviceLevel                            | externalServiceLevel1Code         | n.a.                                | optional                                        | optional                           | optional                                | mandatory                          | optional                            |
+| remittanceInformation Unstructured      | Max140Text                        | n.a.                                | optional                                        | n.a.                               | optional                                | optional                           | optional                            |
+| remittanceInformation Structured        | remittanceInformation Structured7 | mandatory                           | optional                                        | mandatory 6)                       | optional                                | optional                           | optional                            |
+| requestedExecutionDate                  | ISODate                           | mandatory                           | mandatory                                       | mandatory                          | mandatory                               | mandatory                          | mandatory                           |
+| requestedExecutionTime                  | ISODateTime                       | n.a.                                | n.a.                                            | n.a.                               | n.a.                                    | n.a.                               | n.a.                                |
+| intermediaryAgent                       | bicfi                             | n.a.                                | n.a.                                            | n.a.                               | optional 2)                             | n.a.                               | optional 2)                         |
+      
+1) This element may only be used by agreement with the instructed financial institution. currencyOfTransfer is a subfield of equivalentAmount and may only be used if equivalentAmount is used instead of instructedAmount.
```

Comment:
I propose to split this footnote into the two following (needs adaption in table above):
```
1) This element may only be used by agreement with the instructed financial institution.
7) currencyOfTransfer is a subfield of equivalentAmount and may only be used if equivalentAmount is used instead of instructedAmount.
```